web/src/index.css

```diff
@@ -1,3 +1,35 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+@layer base {
+  * { @apply box-border; }
+  
+  body, p, ol, ul, td { @apply font-sans leading-5 m-0 text-gray-800; }
+  
+  a { @apply text-gray-800; }
+  
+  a:hover { @apply bg-green-400; }
+  
+  label { @apply block; }
+  
+  button { @apply m-0; }
+  
+  ul { @apply m-0 p-0; }
+}
+
+
+@layer components {
+  .btn {
+    @apply text-sm p-2 px-8 rounded-full border border-solid bg-white text-gray-800 border border-gray-800 cursor-pointer hover:bg-green-400 no-underline font-semibold;
+  }
+  
+  .card {
+    @apply border border-solid bg-white shadow-sm rounded-sm mx-auto;
+  }
```

Comment:
add comment

Reply:
I pulled these back out into just utilities in each component, but left the button class there as an example to go along with the comment.